hack/release-notes.tmpl

```diff
@@ -0,0 +1,39 @@
+{{with .NotesWithActionRequired -}}
+# Urgent Upgrade Notes
```

Comment:
Please include a markdown file describing how is this used. Ideally we would have RELEASE.md file in the root of the project describing how to handle release notes, how to create releases, ...